ginga/examples/configs/general.cfg

```diff
@@ -96,10 +96,15 @@ icc_working_profile = "sRGB"
 # NOTE: overridden by channel setting of the same name, if any
 scrollbars = 'auto'
 
+# Name of a layout file in ~/.ginga that specifies the UI layout of the
+# program (default is 'layout.json').  This is only used if 'save_layout'
+# (see below) is True
+layout_file = 'layout.json'
```

Comment:
Currently, there is already a `~/.ginga/layout` file. Does that mean `~/.ginga/layout` will be ignored with this patch? If not, what will Ginga do when it finds both `layout` and `layout.json`?

Reply:
If there are both files, Ginga will prefer the `layout.json`.

Reply:
> If there are both files, Ginga will prefer the layout.json

So, with this latest commit, I've reversed course on this one: the `layout` is preferred over `layout.json`. I just think that for this file it's easier to craft Python layouts.

Reply:
Okay, but eventually there should only be one way to do this. Otherwise, it is confusing.